diagram-util/src/main/java/com/powsybl/diagram/util/forcelayout/LayoutAlgorithmRunner.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.diagram.util.forcelayout;
+
+import com.powsybl.diagram.util.forcelayout.geometry.ForceGraph;
+import com.powsybl.diagram.util.forcelayout.geometry.Vector2D;
+import com.powsybl.diagram.util.forcelayout.layouts.AbstractLayoutAlgorithm;
+import com.powsybl.diagram.util.forcelayout.layouts.layoutsparameters.AbstractLayoutParameters;
+import com.powsybl.diagram.util.forcelayout.setup.AbstractSetup;
+import com.powsybl.diagram.util.forcelayout.setup.SetupEnum;
+import com.powsybl.diagram.util.forcelayout.setup.SimpleSetup;
+import com.powsybl.diagram.util.forcelayout.setup.SpringySetup;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.Writer;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.function.Function;
+
+/**
+ * @author Nathan Dissoubray {@literal <nathan.dissoubray at rte-france.com>}
+ */
+public class LayoutAlgorithmRunner<V, E> {
```

Comment:
We should try to make a better difference in classes names between (Abstract)LayoutAlgorithm and LayoutAlgorithmRunner, which does more than just run the LayoutAlgorithm